src/components/MeasurementExplorer.tsx

```diff
@@ -1,6 +1,8 @@
 /** @jsxImportSource @emotion/react */
 import { css } from '@emotion/react';
-import React, { useState } from 'react';
+import { faArrowsLeftRightToLine } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useState } from 'react';
 import { FaExchangeAlt } from 'react-icons/fa';
```

Comment:
Why is there a mix of `react-icons/fa` and a new library?

Reply:
ok. I will open an issue to migrate react-icons/fa to this

Reply:
For now we could rather use this icon: https://fontawesome.com/icons/arrows-left-right?s=solid

And remove { FontAwesomeIcon } from '@fortawesome/react-fontawesome';

Even if are only using react-icons/fa it seems easier to insert.

Reply:
react-icons/fa is easy to insert but we are so limited in icons
there is much more icons in fontawesome

Reply:
I created an issue to find out why it was not updated: https://github.com/react-icons/react-icons/issues/573. In the mean time we can use react-icons and hope that will update soon (or contribute to their project)